Documentation/Signoffs/Create DAQ Subsystem.md

```diff
@@ -0,0 +1,109 @@
+# DAQ Signoff
+
+## Functionality
+
+The primary function of the data acquisition (DAQ) is capturing and displaying relevant testing data in real-time. The DAQ resolves this constraint through a process containing signal conditioners, signal converters, a controller, and software. Through the execution of correctly applying these systems, accurate, real-time data will be derived to support the generation of a model.
+
+## Constraints
+|No.        |Constraints   |Origin   |
+|---|---------------------|--------|
+|3|The DAQ must have at least 10 analog inputs, to cover all possible sensors being used from the sensor integration system.         |Design Constraint |
```

Comment:
I am wondering why constraint 3 says you need 10 analog inputs. You need to detect rpm and torque. Seems like you would need 2 not 10. Why 10?